### ISSUE TITLE
HTTP Method String Support?

### ISSUE THREAD

**Issue by L1lith** [Talha-T/neistion#3]
Just an idea

**Comment by Talha-T** [comment]
Can you explain a little bit more?

**Comment by L1lith** [comment]
Like right now it looks like this in our route config
```
{...
    method: HttpMethod.POST
...}
```
But it could look like
```
{...
    method: "POST"
...}
```

**Comment by Talha-T** [comment]
Actually this can also be strict like an enum. Will be implemented.

**Comment by Talha-T** [comment]
Fixed with 29cd1cb
